src/integrationtests/java/com/aws/greengrass/logmanager/LogManagerTest.java

```diff
@@ -136,6 +137,7 @@ void beforeEach(ExtensionContext context) {
         ignoreExceptionOfType(context, TLSAuthException.class);
         ignoreExceptionOfType(context, InterruptedException.class);
         ignoreExceptionOfType(context, DateTimeParseException.class);
+        ignoreExceptionOfType(context, CrtRuntimeException.class);
```

Comment:
Why have we started ignoring a new exception in a test case?

Reply:
It wasn't happening before; I don't know why. It could be a new exception due to the new SDK version in the nucleus. But this exception doesn't matter, it is only complaining that it isn't configured with certificates, which is fine.